### ISSUE TITLE
Use one function to parse all times

### ISSUE THREAD

**Issue by Anonymous4045** [allthingslinux/tux#420]
Currently, some files like `timeout.py` and `slowmode.py` implement their own ways to parse times in text, such as "10s". This increases the complexity of the files and overall codebase.

Instead, such files should use the already-implemented `convert_to_seconds` utility function.

**Comment by kzndotsh** [comment]
Yeah, those functions are pretty old/early on code and should be redone/standardized and all other functions relating to time in the code base should be re standardized. 

I think a good idea would be to have a specific file dedicated to time like `utils/time.py` as it's a category of functions quite more commonly needed.

**Comment by Anonymous4045** [comment]
Yeah that makes sense. I can try working on this when I get my test bot all set up.

**Comment by anemoijereja-eden** [comment]
In progress, we already have shared time functions in utils/functions.py and are actively designing an improved system for it
